test/configs/windows2.utsc

```diff
@@ -2,12 +2,12 @@
   "testfiles": [
```

Comment:
Could you delete this file. It wrongly resurected

Reply:
done